air/src/lib.rs

```diff
@@ -21,6 +21,7 @@ mod range;
 mod utils;
 use utils::TransitionConstraintRange;
 
+// exported publicly for benchmarking purposes
```

Comment:
Maybe not for this PR, but this looks like a good case for the `internals` feature

Reply:
Agreed!